### ISSUE TITLE
All onmouseover handlers should have an equivalent onfocus handler

### ISSUE THREAD

**Issue by Horlix** [canonical/ubuntu.com#11882]
List of affected URLs: https://docs.google.com/spreadsheets/d/1XsnflMUptvmMu9TKVGiRXiSL1BzPTQFFhgmLnKNWobE/edit#gid=0

Example URLs
- https://ubuntu.com/
- https://ubuntu.com/16-04
- https://ubuntu.com/16-04/azure

Example code:
`<li class="p-navigation__item p-navigation__dropdown-link" role="menuitem" id="enterprise" onmouseover="fetchDropdown('/templates/navigation-enterprise-h', 'enterprise-content'); this.onmouseover = null;">`

**Comment by Horlix** [comment]
Full list of URLs

[All onmouseover handlers should have an equivalent onfocus handler.csv](https://github.com/canonical-web-and-design/ubuntu.com/files/9213445/All.onmouseover.handlers.should.have.an.equivalent.onfocus.handler.csv)

**Comment by lizzochek** [comment]
Hello, @Horlix. Thank you for filing an issue. I can see that currently, unfocus events have been added to the links that have onmouseenter. If you believe this is still incorrect, please reopen the issue wit more details.
